src/test/routes/user/createUser.test.ts

```diff
@@ -0,0 +1,45 @@
+import { describe, test, expect, beforeAll, afterAll } from '@jest/globals';
```

Comment:
```suggestion
import * as dotenv from 'dotenv';
dotenv.config();

import { describe, test, expect, beforeAll, afterAll } from '@jest/globals';
```

Jos ja kun me haluttiin että sovellus populoi envin itse niin varmaan myös testin pitäisi tehdä se?

Olisko tähän joku järkevämpi tapa jestin konffeissa?

Reply:
ENVi on itseasiassa populoitu testejä ajettaessa. Magic yes

Reply:
Sillä siis vaan kun multa puuttui tietokannan root-tunnukset envistä ja testit kaatui siihen

Reply:
Ahaa. No lisäsin sinne testUtilsiin dotenv configin niin varmaan alkaa pelaamaan